src/ZEO/asyncio/tests.py

```diff
@@ -2,17 +2,17 @@
 
 if PY3:
     import asyncio
+
     def to_byte(i):
         return bytes([i])
 else:
-    import trollius as asyncio
+    import trollius as asyncio  # NOQA: F401 unused import
+
     def to_byte(b):
         return b
```

Comment:
```suggestion
from .compat import asyncio

if PY3:
    def to_byte(i):
        return bytes( [i])
else:
    def to_byte(b):
        return b
```

Reply:
master still supports Python 2.7, this must stay.